RiscVPlatformPkg/Universal/Sec/SecMain.c

```diff
@@ -458,6 +493,35 @@ static VOID EFIAPI PeiCore(VOID)
   //
   (*PeiCoreEntryPoint) (&SecCoreData, (EFI_PEI_PPI_DESCRIPTOR *)&mPrivateDispatchTable);
 }
+
+/**
+  RISC-V SEC to PEI handler
+
+  We can't directly transition to PEI because sbi_init() doesn't pass the arguments that the PEI core entry point expects.
+**/
+VOID
+RiscVSecOpenSbiHandler(
+  IN  UINTN HartId,
+  IN  UINTN Foo
+  )
+{
+    DEBUG ((DEBUG_INFO, "Reached %a\n", __FUNCTION__));
+    PeiCore();
+}
+
+VOID
+EFIAPI
+RiscVOpenSbiHartSwitchMode (
+  IN  UINTN   Arg0,
+  IN  UINTN   Arg1,

```

Comment:
Would you like to name these variables more understandable? such as Arg0 -> FucntionArg0  or something

Reply:
Okay